swift_cc_defs.bzl

```diff
@@ -0,0 +1,43 @@
+# Name for a unit test
+UNIT = "unit"
```

Comment:
Bazel has a built in convention for splitting up tests that's similiar to how we distinguish unit vs integration tests.

The convention is `small` vs `large` tests: https://bazel.build/reference/test-encyclopedia#tag-conventions.

The you filter with `bazel test --test_size_filters=small`.

Putting this behind a constant means we can change this to the conventional way if we want without a big refactor.